lib/user-interface/react/src/components/utils.ts

```diff
@@ -170,20 +166,11 @@ export const deleteUserSessions = async (idToken: string) => {
 
 /**
  * Describes all models of a given type which are available to a user
- * @param modelTypes a list of model types to describe
  * @param idToken the user's ID token from authenticating
  * @returns
  */
-export const describeModels = async (
-  modelTypes: ModelTypes[],
-  idToken: string,
-): Promise<DescribeModelsResponseBody> => {
-  const queryParams: string = modelTypes.map((elem) => `modelTypes=${elem}`).join('&');
-  const resp = await sendAuthenticatedRequest(
-    `${RESTAPI_URI}/${RESTAPI_VERSION}/describeModels?${queryParams}`,
-    'GET',
-    idToken,
-  );
+export const describeModels = async (idToken: string): Promise<DescribeModelsResponseBody> => {
+  const resp = await sendAuthenticatedRequest(`${RESTAPI_URI}/v2/serve/models`, 'GET', idToken);
```

Comment:
Similar comment about removing the use of `RESTAPI_VERSION`. If it's going to be removed and just hardcoded you could update the `example_config.yaml` and the CDK code where the `env.js` is created to remove the parameter there too.